cedar-policy/CHANGELOG.md

```diff
@@ -34,13 +24,13 @@ Cedar Language Version: TBD
   without the feature flag. Stabilization comes with changes to fix bugs in the features.
   Level validation is now more permissive when checking `if` expressions (fixing #1507),
   and stricter when checking record literals and entity tag operations (fixing #1505 and #1503). (#1567)
-- Added `Entities::remove_entities()` to remove `Entity`s from an `Entities` struct (resolving #701)
-- Added `PolicySet::merge()` to merge a `PolicySet` into another `PolicySet` struct (resolving #610)
 - Implemented [RFC 53 (enumerated entity types)](https://github.com/cedar-policy/rfcs/blob/main/text/0053-enum-entities.md)  (#1377)
-- Added the experimental feature `tolerant-ast` which allows certain errors to be propogated in AST expressions as an `ExprKind::Error` (#1470)
+- Stabilized [RFC 80 (`datetime` extension)](https://github.com/strongdm/cedar-rfcs/blob/datetime-rfc/text/0080-datetime-extension.md), making `datetime` a deafult feature. (#1541)
```

Comment:
let's link to the accepted RFC text, ie, on `cedar-policy/rfcs/main`, not this one in `strongdm/cedar-rfcs/datetime-rfc`